source/switch_payment_service_provider/switch_psp_to_stripe/index.html.md.erb

```diff
@@ -57,37 +59,45 @@ The live payment is £2.00 and is refundable. Your new Stripe account will be de
 
```

Comment:
Are all numbers supposed to be 1?

Reply:
Yes, this means that the numbers automatically fill out. Makes it easier to add new steps in the middle of the process because you don't need to re-number everything.